.github/workflows/release_and_tag.yml

```diff
@@ -67,4 +73,16 @@ jobs:
         if: inputs.publish_on_pypi
         with:
           user: __token__
-          password: ${{ secrets.PYPI_API_TOKEN }}
\ No newline at end of file
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        if: inputs.publish_on_npm
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@certego'
+
+      - run: npm publish --access public
```

Comment:
Note for Simo of the future: permit to customize options. Eg: access `public` or `restricted`